setup.py

```diff
@@ -36,6 +36,7 @@ def run(self):
         'mapentity==8.0.1',
         'env_file',
         # pinned by requirements.txt
+        'paperclip@git+https://git@github.com/makinacorpus/django-paperclip@add_license_paperclip#egg=paperclip',
```

Comment:
use release

Reply:
If everything is ok here i will do a release in paperclip